jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -212,6 +213,55 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_JITSERVER],
+[
+  AC_MSG_CHECKING([for jitserver])
+  AC_ARG_ENABLE([jitserver], [AS_HELP_STRING([--enable-jitserver], [enable JITServer support @<:@disabled@:>@])])
+  OPENJ9_ENABLE_JITSERVER=false
+
+  if test "x$enable_jitserver" = xyes ; then
+
+    if test "x$OPENJDK_TARGET_OS" != xlinux ; then
+      AC_MSG_RESULT([no (unsupported for $OPENJDK_TARGET_OS)])
+    else
+      AC_CHECK_PROG(PROTOC_INSTALLED,protoc,yes,no)
+      if test "x$PROTOC_INSTALLED" = xno ; then
+        AC_MSG_ERROR([protoc not found])
+      else
+        AC_MSG_CHECKING([for protobuf version])
```

Comment:
We haven't given the `AC_MSG_RESULT` that matches `AC_MSG_CHECKING([for jitserver])` above yet; the log will be a little difficult to follow.